comfy_cli/workflow_ops.py

```
@@ -691,12 +756,40 @@ def _set_widget_impl(
         widget=widget,
         value=value,
         old=old,
+        **extra,
     )
     if warnings:
         op["warnings"] = warnings
     return apply_op(workflow, op, graph), op
 
 
+def _refuse_fatal(warnings: list[dict]) -> None:
+    from comfy_cli.cql.engine import FATAL_FINDING_CODES
```

Pylint (4.0.7)

[convention] Import outside toplevel (comfy_cli.cql.engine.FATAL_FINDING_CODES)

(C0415)